examples/2d/elixir_ape_convergence_test.jl

```diff
@@ -4,10 +4,7 @@ using Trixi
 ###############################################################################
 # semidiscretization of the acoustic perturbation equations
 
-v_mean = (0.5, 0.3)
-c_mean = 1.0
-rho_mean = 1.0
-equations = AcousticPerturbationEquations2D(v_mean, c_mean, rho_mean)
+equations = AcousticPerturbationEquations2D((0.5, 0.3), 1.0, 1.0)
```

Comment:
This is the wrong line to comment on, but please make sure that you change all lines
```julia
ode = semidiscretize(semi, (0.0, 1.0))
```
to something like
```julia
tspan = (0.0, 1.0)
ode = semidiscretize(semi, tspan)
```
Otherwise it is not possible to use `trixi_include(..., tspan=(0, <new_end_time>))` during experimentation. This applies to the other elixirs as well.

Reply:
Done in f7ee3ca